src/main.c

```diff
@@ -18,7 +26,45 @@ static void repl(VM* vm) {
       break;
     }
 
-    interpret(vm, line);
+    int lBraceCount = 0;
+    int rBraceCount = 0;
+
+    for (int i = 0; line[i]; i++) {
+      lBraceCount += (line[i] == '{');
+      rBraceCount += (line[i] == '}');
+    }
+
+    if (lBraceCount > rBraceCount) {
+      char block[1024];
+
+      memcpy(block, line, sizeof(line));
+
+      for (;;) {
+        printf("%.*s ", (lBraceCount - rBraceCount) + 1, ".....");
```

Comment:
Prefer `>` over `.`

Reply:
Also, what is the +1 for?

Reply:
@Calamity210 clone my branch and try it without +1